src/features/pastelID/PastelID.tsx

```diff
@@ -15,6 +19,18 @@ import { openPastelModal } from '../pastelModal'
 import styles from './PastelID.module.css'
 import { createPastelID, fetchPastelIDs } from './pastelIDSlice'
 
+const locatePastelConfDir = () => {
+  if (os.platform() === 'darwin') {
+    return path.join(remote.app.getPath('appData'), 'Pastel')
```

Comment:
Electron doc says that remote is deprecated, has performance and security concerns: https://electronjs.org/docs/api/remote

Can we use suggested ipc instead?

Reply:
Fixed